src/app.js

```diff
@@ -1 +1,58 @@
 'use strict';
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+
+const fs = require('fs');
+const path = require('path');
+
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+readline.question('Enter the path of the file to copy: ', sourceFilePath => {
+  readline.question('Enter the destination directory: ', destinationFolderPath => {
+    readline.close();
+
+    copyFile(sourceFilePath, destinationFolderPath);
+  });
+});
+
+function copyFile(sourceFilePath, destinationFolderPath) {
```

Comment:
Validation checks for source and destination paths should be done before passing them to the copyFile function. Doing so would make the code more modular and easier to test.